events/image-processing-pipeline/gke/README.md

```diff
@@ -0,0 +1,265 @@
+# Image Processing Pipeline - GKE
```

Comment:
While some brief instructions are okay in GitHub, a full tutorial should be published in the official docs or as a community tutorial.